date reported: 2018-08-19
affected users: LordFelix, elektrosmoker
issue: 	- crash using middle/right mouse button on Garage_Mods_Modpack_Button_by_Budyx69
	- left click doesnt open new relhax-chat-window (only if someone created it first)
steps:
---2018-08-19---

-set my standard browser to ms edge (as LordFelix has)
-left click -> opening chat window working! (his error for this is shown below)
-clicked with right/middle-mousebutton on modpackbutton -> CRASH! (nothing logged)

--> crash only happens whith MS Edge set as default browser!!





//---ERRORS-FROM-LordFelix-PYTHON.LOG-----------------------------------------------------------------


//---#1---
2018-08-19 16:15:37.252: INFO: [XFW] Loading mod: com.modxvm.xvm.sounds, v. 7.6.11.8554
2018-08-19 16:15:37.257: ERROR: Traceback (most recent call last):
2018-08-19 16:15:37.257: ERROR:   File "xfw_wwise/python/__init__.py", line 190, in _bank_load
2018-08-19 16:15:37.257: ERROR: RuntimeError: [LoadBank] Cannot load sound bank. Audiokinetic error code: 69 
2018-08-19 16:15:37.257: ERROR: 

2018-08-19 16:15:41.988: ERROR: Traceback (most recent call last):
2018-08-19 16:15:41.988: ERROR:   File "xfw_wwise/python/__init__.py", line 190, in _bank_load
2018-08-19 16:15:41.990: ERROR: RuntimeError: [LoadBank] Cannot load sound bank. Audiokinetic error code: 69 
2018-08-19 16:15:41.990: ERROR: 

// dunno what errors that above are, but
// refered line 190 in "xfw_wwise/python/__init__.py" is the following =>       
//      bank_id = self.native.bank_load_path(unicode(bank_path),unicode(PATH.WOT_RESMODS_DIR + '/audioww/'))


//---#2---left click problem--not joining----
2018-08-19 16:17:23.805: ERROR: [EXCEPTION] (scripts/common/Event.py, 46):
Traceback (most recent call last):
  File "scripts/common/Event.py", line 44, in __call__
  File "scripts/client/messenger/proto/bw/ChannelsManager.py", line 352, in __onSelfEnterChat
ChannelNotFound: Not found a channel with id = -189604, the first request from the server information on the channel with this id

2018-08-19 16:17:23.805: ERROR: Traceback (most recent call last):
2018-08-19 16:17:23.805: ERROR:   File "scripts/client_common/ClientChat.py", line 150, in onChatAction
2018-08-19 16:17:23.805: ERROR:   File "scripts/common/Event.py", line 44, in __call__
2018-08-19 16:17:23.805: ERROR:   File "scripts/client/messenger/proto/bw/ChannelsManager.py", line 352, in __onSelfEnterChat
2018-08-19 16:17:23.805: ERROR: messenger.proto.bw.errors.ChannelNotFound: Not found a channel with id = -189604, the first request from the server information on the channel with this id
//------------------------------------------------------------------------------------------


